.github/workflows/release.yml

```diff
@@ -0,0 +1,45 @@
+name: release
+
+on:
+  release:
+    types: [published, edited]
+
+jobs:
+
+  upload-transifex:
```

Comment:
Can you add a brief comment here that describes the purpose of this job a little bit? Maybe even add a link to the portal where the translation is happening?

Reply:
done